CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Features
 
 ### Fixes
+- `[jest-validate]` Show suggestion only when unrecognized cli param length is greater than 1 ([#10604](https://github.com/facebook/jest/pull/10604))
```

Comment:
```suggestion

- `[jest-validate]` Show suggestion only when unrecognized cli param length is greater than 1 ([#10604](https://github.com/facebook/jest/pull/10604))
```

I think that will make CI happy. You can also run `yarn lint:prettier` locally which will autofix it

Reply:
Yes thank you! I just saw realized I forgot to lint it. Fixing it now.